src/pages/workspaces/workspace/analysis/runtime-common.js

```diff
@@ -11,17 +11,48 @@ import Events from 'src/libs/events'
 import { getLocalPref } from 'src/libs/prefs'
 import { useCancellation, useGetter, useOnMount, usePollingEffect, usePrevious, useStore } from 'src/libs/react-utils'
 import { authStore, azureCookieReadyStore, cookieReadyStore } from 'src/libs/state'
+import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 import { getConvertedRuntimeStatus, usableStatuses } from 'src/pages/workspaces/workspace/analysis/runtime-utils'
 
 
+export const computeStyles = {
+  label: { fontWeight: 600, whiteSpace: 'pre' },
+  value: { fontWeight: 400, whiteSpace: 'pre' },
+  titleBar: { marginBottom: '1rem' },
+  whiteBoxContainer: { padding: '1rem', borderRadius: 3, backgroundColor: 'white' },
+  drawerContent: { display: 'flex', flexDirection: 'column', flex: 1, padding: '1.5rem' },
+  headerText: { fontSize: 16, fontWeight: 600 },
+  warningView: { backgroundColor: colors.warning(0.1) }
+}
+
 export const StatusMessage = ({ hideSpinner, children }) => {
   return div({ style: { paddingLeft: '2rem', display: 'flex', alignItems: 'center' } }, [
     !hideSpinner && spinner({ style: { marginRight: '0.5rem' } }),
     div([children])
   ])
 }
 
+export const RadioBlock = ({ labelText, children, name, checked, onChange, style = {} }) => {
```

Comment:
looks like runtime-common.js has visual components and related bits.... how about we rename it to runtime-common-components.js.

Any small things we can do to get away from large "utils" and "common" dumping grounds the better.